src/App.jsx

```diff
@@ -142,6 +142,9 @@ function App() {
     if (hasCachedProvider()) {
       // then user DOES have a wallet
       connect().then(() => {
+        analytics.identify(address, {
+          wallet: address,
+        });
```

Comment:
What if `analytics` doesn't exist? Do you need the if statement like we added in the Segment Helper?

Reply:
Actually come to think of it there are two other changes you should make quickly:
1. `analytics` isn't declared in `App.jsx` so it won't ever exist anyway
2. move this logic into `useSegementAnalytics` hook as a useEffect that gets called on address change.

Reply:
Great point. Will push this change shortly.